WcaOnRails/app/models/competition.rb

```diff
@@ -128,6 +133,17 @@ class Competition < ApplicationRecord
 
   validates :currency_code, inclusion: { in: Money::Currency, message: proc { I18n.t('competitions.errors.invalid_currency_code') } }
 
+  validates_numericality_of :refund_policy_percentage, greater_than_or_equal_to: 0, less_than_or_equal_to: 100
+  validates :refund_policy_limit_date, presence: true, if: :refund_policy_percentage?
```

Comment:
Does this `if: :refund_policy_percentage?` really do what you want? Isn't `refund_policy_percentage` a number, not a boolean?

Reply:
It does. If `refund_policy_percent` is 0, you can save the form without a `refund_policy_limit_date`; else, the form returns an error.

Reply:
Wow! MySQL stores booleans as integers, so I guess activerecord automatically creates `?` methods for all number columns...